geode-core/src/distributedTest/java/org/apache/geode/alerting/AlertingServiceDistributedTest.java

```diff
@@ -75,8 +74,8 @@
 import org.apache.geode.test.junit.rules.serializable.SerializableTestName;
 
 /**
- * Distributed tests for {@link AlertingService} with {@link DistributedSystemMXBean} in the
- * JMX Manager.
+ * Distributed tests for {@link org.apache.geode.alerting.AlertingService} with {@link
```

Comment:
Does AlertinService need to be fully qualified here?

Reply:
IntelliJ made this change for an unknown reason but ultimately it doesn't hurt anything.

Reply:
Removed FQN